recipe/meta.yaml

```
@@ -26,12 +26,12 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake
     - make
-    - libprotobuf
+    - libprotobuf-static
```

The non-static version is correct here. In `build` we have `libprotobuf` as a requirement to get a runnable version of `protoc` (esp in the case of cross-compiling).
```suggestion
    - libprotobuf
```